### PR TITLE
Quantization fix

### PR DESCRIPTION
**Pull request by gilesknap**
This adds configuration of the quantization interval to scanning.parts.MinTurnaroundPart.

However, two of the tests are failing on Travis but not locally. Have you seen this issue? :
```
_______________________ TestPMACChildPart.test_step_scan _______________________
Traceback (most recent call last):
  File "/home/travis/build/dls-controls/pymalcolm/tests/test_modules/test_pmac/test_pmacchildpart.py", line 517, in test_step_scan
    assert self.child.handled_requests.mock_calls[-1].kwargs['a'] == \
TypeError: tuple indices must be integers or slices, not str
```

### CODE REVIEW

**Comment by gilesknap** [comment]
@thomascobb sub quantization turnaround is now added, with a test.
All tests are passing.

File: malcolm/modules/pmac/velocityprofile.py [hunk 1]
```diff
@@ -64,7 +65,8 @@ def __init__(
             tv2,  # type: float
             a,  # type: float
             v_max,  # type: float
-            settle_time=0  # type: float
+            settle_time=0,  # type: float
+            interval=0.002  # type: float
```
Comment (thomascobb) [inline malcolm/modules/pmac/velocityprofile.py]: Repeating 0.002 seems wrong. Should this be MIN_INTERVAL? Or should there be no default?

Comment (gilesknap) [reply malcolm/modules/pmac/velocityprofile.py]: I have changed the default to be 0 and for that to mean no quantization occurs.
Thus the default reverts to behaviour before the new parameter was added.

**Comment by thomascobb** [comment]
Just to check (can't do the math for the test in my head), if a turnaround takes strictly less than MIN_INTERVAL, then the result is two points exactly MIN_INTERVAL apart.

**Comment by gilesknap** [comment]
Yes you just get a start and end point. See the test at the end of https://github.com/dls-controls/pymalcolm/blob/quantization-fix/tests/test_modules/test_pmac/test_pmacchildpart.py
called test_minimum_profile to verify that it does what you expect.

**Pull request merged**